test/bin/mirror_registry.sh

```diff
@@ -29,23 +29,19 @@ get_container_images() {
 prereqs() {
     mkdir -p "${REGISTRY_ROOT}"
     mkdir -p "${REGISTRY_CONTAINER_DIR}"
-    curl -L https://github.com/quay/mirror-registry/releases/download/v${QUAY_VERSION}/mirror-registry-offline.tar.gz -o /tmp/mirror-registry-offline.tar.gz
-    tar xf /tmp/mirror-registry-offline.tar.gz -C "${REGISTRY_ROOT}"
-    rm -f /tmp/mirror-registry-offline.tar.gz
     sudo dnf install -y podman skopeo jq
+    podman run -d -p 5000:5000 --restart always --name registry "docker.io/distribution/distribution:${DISTRIBUTION_VERSION}"
```

Comment:
We might want to mirror it to our quay, docker.io has pull limits

Reply:
Done